src/App.js

```diff
@@ -1,37 +1,45 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import {
   Button,
   Col,
   ControlLabel,
   Form,
   FormControl,
   FormGroup,
-  PageHeader
+  PageHeader,
 } from 'react-bootstrap';
+import {symbols} from 'currencyformatter.js';
 import 'bootstrap/dist/css/bootstrap.css';
 
 import './App.css';
 import LineItemList from './LineItemList.js';
-import { saveInvoicePDF } from './PDFService.js';
+import {saveInvoicePDF} from './PDFService.js';
+
+const currencyCodes = Object.keys(symbols);
+
+const emptyState = {
+  invoiceNumber: '',
+  fromName: '',
+  imageLogo: null,
+  paymentTerms: '',
+  currency: 'USD',
+  toName: '',
+  date: '',
+  dueDate: '',
+  lineItems: [],
+  notes: '',
+  terms: '',
+};
 
 class App extends Component {
   constructor() {
     super();
-    this.state = {
-      invoiceNumber: '',
-      fromName: '',
-      imageLogo: null,
-      paymentTerms: '',
-      toName: '',
-      date: '',
-      dueDate: '',
-      lineItems: [],
-      notes: '',
-      terms: ''
-    };
+    this.state = emptyState;
 
     this.onFieldValueChange = this.onFieldValueChange.bind(this);
-    this.onLineItemDescriptionChange = this.onLineItemDescriptionChange.bind(this);
+    this.onLineItemDescriptionChange = this.onLineItemDescriptionChange.bind(
+      this
+    );
```

Comment:
This was Prettier again. I suppose the industry trend right now is to just go with Prettier's style suggestions to avoid bikeshedding - how do you feel about doing this?